apps/studio/components/interfaces/Home/ProjectList/ProjectList.tsx

```diff
@@ -150,13 +166,20 @@ export const ProjectList = ({
               <TableHead>Created</TableHead>
             </TableRow>
           </TableHeader>
+          <TableHeader>
```

Comment:
nit: it's not valid HTML to have two `<thead>`s in one table, does it work if we put both `TableRows` in the same `TableHeader`?